### PR TITLE
PAN-OS EDL add logging in case of list deletion

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
- [x] Ready

## Related Issues
https://github.com/demisto/etc/issues/34452

## Does it break backward compatibility?
   - [x] No

### CODE REVIEW

File: Packs/PaloAltoNetworks_PAN_OS_EDL_Management/Integrations/PaloAltoNetworks_PAN_OS_EDL_Management/PaloAltoNetworks_PAN_OS_EDL_Management.py [hunk 1]
```diff
@@ -263,37 +262,44 @@ def edl_update_external_file_command():
...
             md = 'List is empty, deleted from instance context.'
         else:
+            # update list in instance context, can happen upon removal or addition of objects
             dict_of_lists.update({list_name: list_items})
             if verbose:
                 md = tableToMarkdown('List items:', list_items, headers=[list_name])
             else:
                 md = 'Instance context updated successfully.'
 
+    demisto.debug(f'PAN-OS EDL Management updating {list_name} with {len(list_items)} in the integration context.')
```
Comment (yuvalbenshalom) [inline Packs/PaloAltoNetworks_PAN_OS_EDL_Management/Integrations/PaloAltoNetworks_PAN_OS_EDL_Management/PaloAltoNetworks_PAN_OS_EDL_Management.py:302]: @bakatzir you are printing here the len(list_items), but you are not printing if the whole `dict_of_lists` is empty before you save it.

Comment (bakatzir) [reply Packs/PaloAltoNetworks_PAN_OS_EDL_Management/Integrations/PaloAltoNetworks_PAN_OS_EDL_Management/PaloAltoNetworks_PAN_OS_EDL_Management.py:302]: Won't it be printed here?
`demisto.debug(f'PAN-OS EDL Management integration context is empty.')`

File: Packs/PaloAltoNetworks_PAN_OS_EDL_Management/ReleaseNotes/1_0_3.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Palo Alto Networks PAN-OS EDL Management (Deprecated)
+- Internal code improvements.
```
Comment (DeanArbel) [inline Packs/PaloAltoNetworks_PAN_OS_EDL_Management/ReleaseNotes/1_0_3.md:4]: Maybe - Maintenance and stability enhancements.?

**Review by DeanArbel — APPROVED** [review]
Nice

**Comment by bakatzir** [comment]
related to: https://github.com/demisto/etc/issues/37940

**Pull request merged**